ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -491,6 +491,14 @@ object Config {
             "Enable preview version of the next Daml-LF language. Should not be used in production."
           )
 
+        opt[Unit]("stable-only")
+          .optional()
+          .hidden()
+          .action((_, c) => c.copy(engineMode = EngineMode.StableOnly))
+          .text(
+            "Disable the loading of unstable LF packages outside of the current LF stable version. Should not be used in production."
+          )
+
```

Comment:
Why do we want to change the behavior of KV ?

Reply:
This is the parser that ledger-on-memory uses. The point is to be able to test without loading old (unstable) packages.

Reply:
OK the purpose is test. In this case I do not think  we need the constant `EngineMode.StableOnly`, we can defined it manually here.

I am a bit concern of changing the code of ledger-on-memory, to test a feature that will be used by canton only.
I think having a unit test of the Compiler and ask canton to add a test to be sure they refuse loading old LF version is enough.

Reply:
I don't understand what you mean by "define it manually here". Could you elaborate?

Reply:
In  `ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala` you could write : 
```
     case EngineMode.StableOnly => LanguageVersion.Stable.copy(max = LanguageVersion.Stable.max)
```

Reply:
Yes, but that still requires defining `EngineMode.StableOnly`?

Reply:
Sorry for only getting to this so late. How about adding a `min-lf-version` flag that allows you to set the minimum required LF version? That way we can set it to whatever we want in tests, we don’t need any constants in `Engine` and we don’t have to debate whether we call it latest, stable or latest stable?

Reply:
and you can replace `EngineMode` by `VersionRange[LanguageVersion]`.

Reply:
This is a separate issue. 

I think we should not have  `EngineMode.StableOnly` neither because I do not think we should change the code of KV for testing a feature that will be used by Canton only, but : 
- I care less 
- It should be approved by @miklos-da teams.

Reply:
It’s sort of related since a `min-lf-version` flag requires representing some form of range (or at least a lower bound) and at that point the redundancy between a new constructor to `EngineMode` and the existing constructors seems to do more harm than good. But yeah, I don’t care all that much either.

Reply:
@moritzkiefer-da > This is a separate issue.

Reply:
Despite the concern I express before, I will not veto changes in KV code. 
By the way I slightly prefer the `min-lf` flag.

Reply:
I added a `--daml-lf-min-version-1.14` flag in lieu of a full version parser.